pkg/image/registry/imagesignature/strategy.go

```diff
@@ -0,0 +1,74 @@
+package imagesignature
+
+import (
+	"crypto/sha1"
+	"fmt"
+
+	"github.com/golang/glog"
+
+	kapi "k8s.io/kubernetes/pkg/api"
+	"k8s.io/kubernetes/pkg/runtime"
+	"k8s.io/kubernetes/pkg/util/validation/field"
+
+	imageapi "github.com/openshift/origin/pkg/image/api"
+	"github.com/openshift/origin/pkg/image/api/validation"
+)
+
+// strategy implements behavior for ImageStreamTags.
+type strategy struct {
+	runtime.ObjectTyper
+}
+
+var Strategy = &strategy{
+	ObjectTyper: kapi.Scheme,
+}
+
+func (s *strategy) NamespaceScoped() bool {
+	return false
+}
+
+func (s *strategy) PrepareForCreate(obj runtime.Object) {
+	signature := obj.(*imageapi.ImageSignature)
+
+	signature.Conditions = nil
```

Comment:
You're going to need a fuzz tests against this method to make sure that all fields except the whitelisted ones are removed.